python/paddle/fluid/tests/unittests/test_power.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import paddle
+import paddle.tensor as tensor
+import paddle.fluid as fluid
+from paddle.static import Program, program_guard
+import numpy as np
+import unittest
+
+
+def _run_power(mode, x, y):
+    # dynamic mode
+    if mode == 'dynamic':
+        # paddle.enable_imperative()
+        paddle.disable_static()
+        # y is scalar
+        if isinstance(y, (int, float)):
+            x_ = paddle.to_variable(x)
+            # print(x_)
+            y_ = y
+            res = paddle.power(x_, y_)
+            return res.numpy()
+        # y is tensor
+        else:
+            x_ = paddle.to_variable(x)
+            y_ = paddle.to_variable(y)
+            res = paddle.power(x_, y_)
+            return res.numpy()
+    # static mode
+    elif mode == 'static':
+        # paddle.disable_imperative()
+        paddle.enable_static()
+        # y is scalar
+        if isinstance(y, (int, float)):
+            with program_guard(Program(), Program()):
+                x_ = paddle.data(name="x", shape=x.shape, dtype=x.dtype)
+                y_ = y
+                res = paddle.power(x_, y_)
+                place = fluid.CPUPlace()
+                exe = fluid.Executor(place)
+                outs = exe.run(feed={'x': x}, fetch_list=[res])
+                return outs[0]
+        # y is tensor
+        else:
+            with program_guard(Program(), Program()):
+                x_ = paddle.data(name="x", shape=x.shape, dtype=x.dtype)
+                y_ = paddle.data(name="y", shape=y.shape, dtype=y.dtype)
+                res = paddle.power(x_, y_)
+                place = fluid.CPUPlace()
+                exe = fluid.Executor(place)
+                outs = exe.run(feed={'x': x, 'y': y}, fetch_list=[res])
+                return outs[0]
+
+
+class TestPowerAPI(unittest.TestCase):
+    """TestPowerAPI."""
+
+    def test_power(self):
+        """test_power."""
+        np.random.seed(7)
+        # test 1-d float tensor ** float scalar
```

Comment:
It is fine to me here, but just a knowledge let you know.

1. In python you can write
```
def test_1d_float_tensor_pow_float_scalar
```
Then you don't need the `print("-----test1----")` and the comment. Python trackback will tell you which test is wrong.

2. If you are doing this kind of test in the future. `print("----test 1-d float tensor ** float scalar ----")` is even better then `print("----test1----")`